src/js/tech/html5.js

```diff
@@ -1235,6 +1235,20 @@ Html5.resetMediaElement = function(el) {
   'autoplay',
 
   /**
+   * Get the value of `playsinline` from the media element. `playsinline` indicates
+   * that the media should play inline in iOS Safari.
```

Comment:
iOS is an example of a browser that will only play inline if the attribute is present.